saas/data_plane/kubernetes/tenant_chart/values.yaml

```diff
@@ -0,0 +1,14 @@
+image: docker.io/civiform/civiform
+imageTag: latest
+namespace: tenant-bham
+nodePoolName: np-tenant-bham
+tenantId: bham
+ksaName: sa-tenant-bham
+gsaEmail: civiform-tenant-sa-bham@bion-sandbox.iam.gserviceaccount.com
+dbUsername: civiform-tenant-sa-bham@bion-sandbox.iam
+dbConnectionName: bion-sandbox:us-central1:civiform-tenant-bham
+publicFQDN: bham.tenant.civiform.dev
+sslCertName: tenant-civiform-dev
+envVarConfigMapName: tenant-bham-server-env-vars
+envVarSecretName: tenant-bham-server-secrets
+staticIPName: tenant-bham-static-ip
```

Comment:
(question) Are any of these things we don't want exposed to the internet?

Reply:
Probably best not to leak them in prod, but no, no secrets are passed to Helm. The overall approach I've attempted here:

1. use IAM auth wherever possible, which eliminates passwords as an attack vector in those places
2. don't pass secrets through TF, which keeps them out of TF state files
3. k8s prod secrets should always be stored in k8s [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) resources and then attached to the resources that need them (e.g. `envVarSecretName` here)